Makefile

```diff
@@ -125,6 +125,9 @@ docker-push:
 	docker push ${DOCKER_IMG}:${GIT_VERSION}
 	docker push ${DOCKER_IMG}:latest
 
+.PHONY: imports
+imports:
+	goimports --local github.com/kudobuilder -w .
```

Comment:
what do you think about rather keep relying on one tool and one makefile command (make lint) and just enabling goimports linter in golangci-lint

https://github.com/golangci/golangci-lint/blob/master/vendor/github.com/golangci/gofmt/goimports/goimports.go

https://github.com/golangci/golangci-lint

I did not test it but based on the docs it should be possible so feel free to tell me I am wrong :)

I see that there are some shortcomings of golangci-lint like you cannot pass parameters to linters :( https://github.com/golangci/golangci-lint/issues/646

In any case I think we should at least try to install goimports before running it as we do we golangci lint if we decide that we need to use it explicitly

Reply:
Yes, `golanci-lint` already provides `goimports`. It can be configured like this by adding
```
linters-settings:
  goimports:
    local-prefixes: github.com/kudobuilder/kudo
```
to `.golanci.yml`.

Reply:
in agreement with modifying the linter... however this is not that.   Linters don't modify code... this was a useful execution to modify code.   If we don't want it here... I'm happy to remove it.